### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by rafpaf** [webdiscus/pug-loader#21] [General]
This works beautifully - I'm very grateful!

**Comment by webdiscus** [comment]
@rafpaf 

If you like the pug-loader, you might also be interested in the [pug-plugin](https://github.com/webdiscus/pug-plugin).

The pug-loader is designed for usage with Vue oder Angular.
The pug-plugin is very powerful Webpack plugin for usage Pug with styles and scripts w/o Vue, etc.

Thanks for usage.
